### PR TITLE
Use abbreviated months in German short time format

### LINKED ISSUE #1062: Abbreviated months in short time format for German
The short time format in German is `%d. %B, %H:%M Uhr` (see [de.yml](https://github.com/svenfuchs/rails-i18n/blob/ad26c50e1fea0eac2c9b85815f2c051f33a78045/rails/locale/de.yml#L224)) which produces a string like `28. Dezember, 23:23 Uhr`. So currently, the names of the months are not abbreviated.

Could we switch to the abbreviated format instead, using `%d. %b, %H:%M Uhr` (note the lowercase `b`)? This woud produce a shorter string like `28. Dez, 23:23 Uhr`?

### PR DESCRIPTION
**Pull request by pirlande-fw**
Fix #1062: use abbreviated months in the short time format for all German languages.

### CODE REVIEW

**Review by sunny — APPROVED** [review]
Can you add an entry to the CHANGELOG ? 🙏🏻

**Comment by pirlande-fw** [comment]
> Can you add an entry to the CHANGELOG ? 🙏🏻

Sure! It's done

**Comment by pirlande-fw** [comment]
Hi @sunny, do you know if a maintainer can approve this PR at some point?

**Comment by pama** [comment]
I'm concerned with this change because it will eventually change the result for others. Are you proposing this because it is how it's done in Germany, or because you prefer this way?

**Comment by pama** [comment]
Hi @pirlande-fw thank your for your feedback. I don't know how reliable that website is, as the Portuguese reference seems incorrect ([As abreviaturas dos meses dos anos segundo o novo Acordo Ortográfico](https://ciberduvidas.iscte-iul.pt/consultorio/perguntas/as-abreviaturas-dos-meses-dos-anos-segundo-o-novo-acordo-ortografico/30221))

But I agree with you that they seem inconsistent from the rails-i18n point of view.

I will wait a few days to see if we get more feedback before merging this.

Thanks for your contribution!

**Comment by pirlande-fw** [comment]
Hi @pama, have we made some decision on including this change?

**Comment by pama** [comment]
@pirlande-fw could you fix the conflict? I will merge afterward.

**Comment by pirlande-fw** [comment]
> @pirlande-fw could you fix the conflict? I will merge afterward.

Sure it's done

**Pull request merged**
